src/components/Documentation/index.tsx

```diff
@@ -22,7 +25,7 @@ const Documentation: React.FC<IDocumentationProps> = ({
   headings
 }) => {
   const { source, prev, next, tutorials } = getItemByPath(path)
-  const githubLink = `https://github.com/iterative/dvc.org/blob/master/content${source}`
+  const githubLink = getGithubLink(source)
```

Comment:
not too sure about purpose of this change but whatever...

Reply:
We need to use the GitHub link in two places now. The component that renders markdown and the component that renders JSX. To avoid using the link twice I just created a function that holds the link  and both components use the function. I can take it off if you like 😀

Reply:
Afaik you're only using the function in one place in this file... oh you mean you're using it elsewhere from this file?

Reply:
Ah I see yes you are...